help.c

```diff
@@ -618,7 +621,19 @@ const char *help_unknown_cmd(const char *cmd)
 				   _("Continuing under the assumption that "
 				     "you meant '%s'."),
 				   assumed);
-		else {
+		else if (autocorrect == AUTOCORRECT_PROMPT) {
+			char answer[3];
+			if (!isatty(STDIN_FILENO) | !isatty(STDERR_FILENO))
+				exit(1);
+			fprintf_ln(stderr,
+				   _("Assuming you meant: '%s'.\n"
+				     "Continue? (y/n) "),
+				   assumed);
+			if (!fgets(answer, sizeof(answer), stdin))
```

Comment:
I seem to remember that we spent some time to consolidate similar code patterns to use a single function instead (was it a function declared in `prompt.h`? I don't remember... *clicketyclick* ah, it was `git_prompt()`, I think). It appears to me that we want to use that function here, too.